### PR TITLE
added filterpercepts action and filter indicators in `DUWO.mod2g`

### PR DESCRIPTION
**Pull request by Denpeer**
Self explanatory.

User Story:
As a developer I want to be able to disable certain percepts so that I can test my goal agent code without interference from the tygron environment.

### CODE REVIEW

**Comment by Denpeer** [comment]
I'll resolve the conflicts

File: src/DUWOagent/DUWOagent.mas2g [hunk 1]
```diff
@@ -1,5 +1,5 @@
 use "../../dependencies/contextvh-1.0.1-jar-with-dependencies.jar" as environment 
-	with project = "vhproject_duplicate", stakeholders = ["DUWO"].
+	with project = "vhproject_highbud", stakeholders = ["DUWO"].
```
Comment (klikooo) [inline src/DUWOagent/DUWOagent.mas2g]: Why change the map?

Comment (Denpeer) [inline src/DUWOagent/DUWOagent.mas2g]: at the moment when i created the branch duplicate didn't have enough budget to be properly tested. I'll change it

**Comment by klikooo** [comment]
I approve when travis passes

**Comment by MarcianoJ** [comment]
Looks good, travis passed, I approve

**Pull request merged**